### DISCUSSION TITLE
Add rc / dotfile support for default settings

### DISCUSSION THREAD

**Discussion by Nightfirecat** [boyter/scc#341] [Ideas]
Some users may have settings which they always want to use, such as `--no-cocomo`, or others, based on use case or personal preference. Presently that's only possible by always passing the setting with every invocation. It would be preferable to be able to create a `.sccrc`-type file in the home directory or XDG config directory or some other location of the sort to always pass these options by default.

**Comment by boyter** [comment]
Not a bad idea. Have an example of someone else who has implemented this that you like?

**Reply by Nightfirecat** [reply]
[ripgrep](https://github.com/BurntSushi/ripgrep) has a system which is pretty simple: set the `RIPGREP_CONFIG_PATH` environment variable (for me, `~/.ripgreprc`) and create a file containing the shell arguments to be passed by default to each invocation of the command: https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#configuration-file

For instance, my config file is as follows:
```
$ cat ~/.ripgreprc 
# https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#configuration-file

# Limit output width
--max-columns=150

# Print Unix-style paths
--path-separator=/

# Search hidden files and directories
--hidden

# Don't search .git directories
--glob=!.git
```
